### ISSUE TITLE
Release Package

### ISSUE THREAD

**Issue by mikemhenry** [OpenFreeEnergy/kartograf#2]
- [x] pypi release
- [x] conda-forge release https://github.com/conda-forge/staged-recipes/pull/22026

**Comment by mikemhenry** [comment]
I can help with the conda-forge release once the pypi one is done @richardjgowers if you want to devide and conquer

**Comment by richardjgowers** [comment]
Pypi is done already

**Comment by mikemhenry** [comment]
Cool I thought it was just parked.

**Comment by mikemhenry** [comment]
Tracking conda-forge release here: https://github.com/conda-forge/staged-recipes/pull/22026

**Comment by mikemhenry** [comment]
:shipit: :tada:
